pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+requires = [
+  "setuptools>=51.0",
+  "wheel>=0.36",
```

Comment:
Fun fact: `setuptools.build_meta` already declares `wheel` as a dependency internally so you actually never need to add it to `setuptools`.

Reply:
Doesn't really matter IMHO, isn't it?

Reply:
It's just one thing less to maintain I recently noticed jaraco removing it actively from his projects, for example. I'll probably update the setuptools docs not to have it in the example too.

Reply:
Got you, thanks. 
`wheel` is almost 'frozen' library, it is updated very rarely though.

Reply:
Done on the master

Reply:
_(https://github.com/pypa/setuptools/pull/3056)_